src/modules/dashboard/components/ActionsPage/DetailsWidget/DetailsWidget.tsx

```diff
@@ -153,6 +165,34 @@ const DetailsWidget = ({
           </div>
         </div>
       )}
+      {values?.isAddSafe && (
```

Comment:
Also, you need to change.  Add the details of this action and the action type itself to the `DETAILS_FOR_ACTION` mapping that we have, that's something that we do for every action. Then each detail should have its own separate conditional like the rest of the details in this component:

```
...
{detailsForAction.Chain && (...)}
{detailsForAction.Safe && (...)}
...
```

This is really useful and important because the component stays adaptable to any other action that might share details with this one.

Reply:
Overall, take a look at the `staticMaps` file and make sure that this new action type is correctly integrated just like the other action types that we have